websites/O/Oneanime/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "OxParadox",
+		"id": "957329295603269652"
+	},
+	"service": "OneAnime",
+	"description": {
+		"en": "Take advantage of our new OneAnime platform to watch your favorite anime totally free, in VF and VOSTFR!",
+		"fr": "Profiter de notre nouvelle plateforme OneAnime pour regarder vos animés préféré totalement gratuitement, en VF et VOSTFR!"
+	},
+	"url": [
+		"oneanime.fr",
+		"www.oneanime.fr"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/NLrk8Qp.png",
```

Comment:
Logos should be at least [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) large. Find a larger version or upscale it using something like waifu2x.